packages/jobs/src/job-definitions/batchEvaluations/runBatchEvaluationJob.ts

```diff
@@ -43,7 +47,7 @@ export const runBatchEvaluationJob = async (
   if (!workspace) throw new NotFoundError('Workspace not found')
 
   const commit = await new CommitsRepository(workspace.id)
-    .find(document.commitId)
```

Comment:
I don't get this, how can document.commitid be different than commitUuid? the documentversion always poins to its commit no?

Reply:
Before it was the commitId the document was created (live). Now is the commit that the user is seeing. live or a draft commit. So, the logs are attached to that commit. Which is what we want